src/components/molecules/SearchAutocomplete/index.tsx

```diff
@@ -18,6 +19,7 @@ const SearchAutocomplete: FC<SearchAutocompleteProps> = ({ setTag }) => {
   const [showSuggestions, setShowSuggestions] = useState(false);
   const [searchTerm, setSearchTerm] = useState("");
   const [showClearButton, setShowClearButton] = useState(false);
+  const [suggestionType, setSuggestionType] = useState("");
```

Comment:
I think we don't need additional state here (`[suggestionType, setSuggestionType]`), I'd set tag only when suggestion id of type TAG and therefore whenever we check whether tag exists we can assume the suggestion is of type TAG -> we use our fake input with proper styling. I have checked it locally and works good :)